src/js/tech/tech.js

```diff
@@ -322,6 +322,7 @@ class Tech extends Component {
     }
 
     let textTracksChanges = Fn.bind(this, function() {
+      let tracks = this.textTracks();
       let updateDisplay = () => this.trigger('texttrackchange');
```

Comment:
I'm not totally sure this will be removed in the loop below since the function is created anew (I believe) every invocation of this method. That would be a pre-existing bug-- just thought it's worth mentioning.